internal/node/node.bzl

```diff
@@ -458,13 +433,6 @@ jasmine_node_test(
         `--node_options=--preserve-symlinks`
         """,
     ),
```

Comment:
is it still the case that you can write `--node_options` into the params file? I suppose not because we don't read it at all before it is read in the node program that's already running.

I don't think anyone needed to spill node_options into the file (though I suppose if you need to bootstrap a hundred files you'll run into argv limit... you could solve that some other way like bundle up your bootstraps)

Reply:
good point. I'd can't think of why someone would want to --require 100 files but stranger things have happened. bundling would work in that case. 👍